src/components/Alerts/PlaybookToastAlerts.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Alert,
+  AlertGroup,
+  AlertActionCloseButton,
+} from '@patternfly/react-core';
+
+const PlaybookToastAlerts = ({
+  key,
+  title,
+  description = '',
+  variant = 'success',
+}) => {
+  const [activeAlerts, setActiveAlerts] = useState([]);
+
+  useEffect(() => {
+    console.log('Testing the changes on active alert: ', key);
+    console.log(title);
+    addActiveAlert(key, title, description, variant);
+  }, [key]);
+
+  const removeAlert = (key) => {
+    console.log('Testing that weve cleared out alerts');
+    setActiveAlerts([...activeAlerts.filter((alert) => alert.key !== key)]);
```

Comment:
No need to create an additional copy of the array (the [...]). `Array.prototype.filter` already returns new copy.

How ever you should use the set callback instead to safely update the value:
```jsx
setActiveAlerts(prev => prev.filter((alert) => alert.key !== key))
```